From 69563d90e2f51ccffef2067ece678cde782146ad Mon Sep 17 00:00:00 2001
From: "Srivatsa S. Bhat (VMware)" <srivatsa@csail.mit.edu>
Date: Fri, 31 Mar 2023 13:56:48 -0700
Subject: [PATCH] linux, Makefile: Add kernel flavor info to the generated
 version.h file

Add "PHOTON_KERNEL_FLAVOR_GENERIC" macro definition to the generated file
include/generated/uapi/linux/version.h to identify Photon's linux
(generic) kernel flavor.

Signed-off-by: Srivatsa S. Bhat (VMware) <srivatsa@csail.mit.edu>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index fa5f08d5e76d..5a5a5393e08c 100644
--- a/Makefile
+++ b/Makefile
@@ -1260,7 +1260,7 @@ define filechk_version.h
 	echo '#define PHOTON_KERNEL';                                                   \
 	echo '#define PHOTON_KERNEL_RELEASE                                             \
 	$(shell echo -n "$(KERNELRELEASE)" | cut -d'-' -f2 | cut -d'.' -f1)';           \
-	echo '#define PHOTON_KERNEL_FLAVOR_'
+	echo '#define PHOTON_KERNEL_FLAVOR_GENERIC'
 endef
 
 $(version_h): private PATCHLEVEL := $(or $(PATCHLEVEL), 0)
-- 
2.35.6

